.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,10 @@
 
 ## What is the value of this and can you measure success?
 
+<!--
+If setting up an AB test, make sure you have read our [AB testing doc](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md).
```

Comment:
People may have been unable to set up an AB test without reading that doc in the first place; is it worth having a final section of the AB README that summarises things to double check (Including the new addition to the README)?